trussworks/react-uswds
# chore: Update to USWDS 2.11.2

# Summary

This PR updates USWDS to version 2.11.2 and makes minor, non-functional changes to existing components. Any other changes needed to match 2.11 are documented in further issues below.

Updates included are:
- 2.11.0 [release notes](https://github.com/uswds/uswds/releases/tag/v2.11.0) & [code diff](https://github.com/uswds/uswds/commit/e06173bd4be19d48acf0169800a1305858e35601#)
    - updated some static (non-JS) component markup based on new USWDS component code
- 2.11.1 [release notes](https://github.com/uswds/uswds/releases/tag/v2.11.1) & [code diff](https://github.com/uswds/uswds/commit/af0bc5c0cad15540095f087b6de590178aee3912)
    - no changes needed
- 2.11.2 [release notes](https://github.com/uswds/uswds/releases/tag/v2.11.2) & [code diff](https://github.com/uswds/uswds/commit/de44902441415952ae313418a566a06d7867a789)
    - updated thumbs down icon asset

## Related Issues or PRs

Followup issues for 2.11 updates (not included in this PR):
* https://github.com/trussworks/react-uswds/issues/1233
* https://github.com/trussworks/react-uswds/issues/1234
* https://github.com/trussworks/react-uswds/issues/1235
* https://github.com/trussworks/react-uswds/issues/1541
    * 2.11 includes some updates to the implementation of the Tooltip component that we should roll out in our version if appropriate
* https://github.com/trussworks/react-uswds/issues/1618
* https://github.com/trussworks/react-uswds/issues/1619
* https://github.com/trussworks/react-uswds/issues/1620

Closes #1304 

## How To Test

* Verify that you are able to use this branch alongside an application using USWDS 2.11 without any issues.
